src/bin/tikv-ctl.rs

```diff
@@ -824,20 +828,23 @@ fn main() {
                 .about("print the range db range")
                 .arg(
                     Arg::with_name("from")
+                        .required(true)
                         .short("f")
                         .long("from")
                         .takes_value(true)
                         .help("set the scan from raw key, in escaped format"),
                 )
                 .arg(
                     Arg::with_name("to")
+                        .conflicts_with("limit")
```

Comment:
why `limit` conflicts with `to`?  I think getting limited items in `[from, to)` is reasonable.